src/package/packageVersionReport.ts

```diff
@@ -39,6 +50,22 @@ const getLogger = (): Logger => {
   return logger;
 };
 
+function constructQuery(connectionVersion: number, verbose: boolean): string {
+  // Ensure we only include the async validation property for api version of v60.0 or higher.
+  let queryFields = connectionVersion > 60 ? [...defaultFields, 'ValidatedAsync'] : defaultFields;
+  verboseFields = connectionVersion > 60 ? [...verboseFields, 'EndToEndBuildDurationInSeconds'] : verboseFields;
```

Comment:
I think you need to include ValidatedAsync as well as EndToEndBuildDurationInSeconds for verbose fields

Reply:
From what I remember Dileep mentioned `ValidatedAsync` will have same behavior as `Validation Skipped` and the latter one is non-verbose field.

Reply:
Misread the code, looks okay.